### DISCUSSION TITLE
Copilot has free support for people with disabilities, while they are programmers?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20092] [Copilot Conversations]
I am a student (disabled person) learning to code, currently I am using Copilot for free for 2 months, almost expired, for me 10$/month is a large amount, while I am a student, No stable job, currently studying.
I look forward to receiving a positive response from github. Thanks!

**Comment by rauzerfabiano** [comment]
For now what we have is the following:

Not all students get the free access, only the students who have access to the Github Student Pack and this will not be valid anymore after they graduate.

**Comment by Zoltus** [comment]
Ahh, havent heard anything about that. I mean ye for sure it should be free for disabled people if its free for students. Not everyone can afford for 10$/month.

You should apply for student licence here just incase:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack
I hope u get the licence<3

**Reply by ghost** [reply]
Cảm ơn lời khuyên của bạn

**Reply by Zoltus** [reply]
> Cảm ơn lời khuyên của bạn

Haha I see you have the pro now:)? Also if I answered your question u can mark my comment as answered:)
